components/Cards/examples/CardExampleViewController.swift

```diff
@@ -40,6 +40,8 @@ class CardExampleViewController: UIViewController {
     shapeLayer.path = bezierPath.cgPath
     imageView.layer.mask = shapeLayer
 
+    let colorScheme = MDCSemanticColorScheme()
+    MDCCardsColorThemer.applySemanticColorScheme(colorScheme, to: card)
```

Comment:
Would you add an example to Dragons where you apply a non-default scheme to the cards?